test/unit/spec/v2/models/AppState_spec.js

```diff
@@ -25,21 +25,21 @@ describe('v2/models/AppState', function() {
       this.initAppState({ idx: { actions: { cancel: 'invalid' } } });
       expect(this.appState.shouldShowSignOutLinkInCurrentForm()).toBe(false);
     });
-    it('returns false if the currentFormName is in FORMS_WITHOUT_SIGNOUT', () => {
-      this.initAppState({ idx: { actions: { cancel: () => {} } } }, FORMS_WITHOUT_SIGNOUT[0]);
-      expect(this.appState.shouldShowSignOutLinkInCurrentForm()).toBe(false);
+    it('returns false if param hideSignOutLinkInMFA is true but there is no remediation action', () => {
```

Comment:
this is a double criteria to return false, right? hideSignOutLinkInMFA=true and no actions.
Not sure if that was the intention, or if we should have one more for hideSignOutLinkInMFA=false and no actions

Reply:
This should actually be `false`, as we'd never execute the check for remediation actions unless `hideSignOutLinkInMFA` is set to `false`. Updating!